src/odbc/unittests/connect.c

```diff
@@ -78,70 +88,59 @@ TEST_MAIN()
 		return 0;
 	}
 
-	/* try connect string with using DSN */
-	printf("connect string DSN connect..\n");
+	/* try connect string with using DSN= (should behave the same as SQLConnect)
+	 */
+	printf("SQLDriverConnect(DSN=%s;...)\n", common_pwd.server);
 	init_connect();
 	sprintf(tmp, "DSN=%s;UID=%s;PWD=%s;DATABASE=%s;", common_pwd.server,
 		common_pwd.user, common_pwd.password, common_pwd.database);
-	CHKDriverConnect(NULL, T(tmp), SQL_NTS, (SQLTCHAR *) tmp, sizeof(tmp)/sizeof(SQLTCHAR), &len, SQL_DRIVER_NOPROMPT, "SI");
+	/* note: final argument "SI" means CHKDriverConnect will exit() if the result is not Success or Sucess With Info */
```

Comment:
these strings are used everywhere, I would say it's base knowledge, the comment looks redundant to me